### DISCUSSION TITLE
Integrating measurements

### DISCUSSION THREAD

**Discussion by jonnor** [Universite-Gustave-Eiffel/NoiseModelling#352] [Q&A]
Hi,
it seems possible to simulate industrial noise outdoors (using point sources) with NoiseModelling. I was wondering if it possible to also also incorporate measurements at different receiver points into the model, such that these are used to adjust/correct the model?

I am aware that this might be a research-level type question more than a howto or feature request. So if anyone has tips about research papers or techniques that might be applicable, that would also be much appreciated.

**Comment by alesieur** [comment]
Hi, 
I just finished my PhD on data assimilation for noise mapping techniques, on which I used NoiseModelling for this specific suject, more specifically for traffic noise. The technique I used was the generation of a meta-model in order to emulate the behavior of the noise map, and then I have used two different data assimilation methods: BLUE and Inverse modelling. You can check the three related articles I have published on this topic:
- Meta-modeling for urban noise mapping (JASA, https://doi.org/10.1121/10.0002866 )
- Data assimilation for urban noise mapping with a meta-model (Applied Acoustics, https://doi.org/10.1016/j.apacoust.2021.107938)
- Inverse modeling and joint state-parameter estimation with a noise mapping meta-model (JASA, https://doi.org/10.1121/10.0004984)
Feel free to contact me if you have any questions regarding this topic

**Reply by jonnor** [reply]
Hi @alesieur - thank you for the response. Those papers sound very interesting, I will have a look at them
